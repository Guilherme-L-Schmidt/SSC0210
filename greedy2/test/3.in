5
6 6
WWWWWW
WWWWWW
WWWWWW
WWWWWW
WWWWWW
WWWWWB
6 6
WWWWWW
WWWWWW
WWWWWW
WWWWWW
WWWWWW
WWWWWW
10 7
WWWWWWW
WWBWWWB
WWWWWWW
WWWWWWW
BWWWWWB
WWWBWWW
WWWBWWW
WWWWWWW
WWWWWWB
WWWWWWW
9 10
WWWWWWBWWW
WWWWWWWBWW
WWWWWWWWWW
WWBWWWWWWW
WWWWWWWWWW
WWWWWBBWWB
WWWWWWWWWW
WWWWWWWWWW
WBWWWWBBWW
10 10
WWWWWWWWWW
WWWWWWWWBW
WWWWWWWWWB
WWWWWBWWWB
WWWBWWWWWW
WWBWWBBWWW
WWWWBWWBWW
WWWWWWWWBW
WWWWWWWWWW
WWWWBWWWBW
