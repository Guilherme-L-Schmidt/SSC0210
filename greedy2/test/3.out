1 1
0 0
9 7
9 9
9 8
